app/lib/common/widgets/html_editor/html_editor.dart

```diff
@@ -370,7 +372,7 @@ class HtmlEditorState extends State<HtmlEditor> {
           // widget pass through
           editable: widget.editable,
           shrinkWrap: widget.shrinkWrap,
-          autoFocus: widget.autoFocus,
+          autoFocus: true,
```

Comment:
keeping auto-focus  `true` for accessibility on desktop.

Reply:
but shouldn't this be this then?
```suggestion
          autoFocus: isDesktop,
```

I don't think we want to open this on mobile every time as well...

Reply:
> but shouldn't this be this then?
> 
> I don't think we want to open this on mobile every time as well...

Yes, we don't and we don't have to put ```isDesktop``` check here as it only belongs in `desktopEditor()` method:
```
Widget desktopEditor(String? roomId) {
    return FloatingToolbar(
      items: [
        paragraphItem,
        ...headingItems,
        ...markdownFormatItems,
        quoteItem,
        bulletedListItem,
        numberedListItem,
        linkItem,
        buildHighlightColorItem(),
      ],
      textDirection: _textDirection,
      editorState: editorState,
      editorScrollController: editorScrollController,
      style: FloatingToolbarStyle(
        backgroundColor: Theme.of(context).colorScheme.surface,
        toolbarActiveColor: Theme.of(context).colorScheme.tertiary,
      ),
      child: Directionality(
        textDirection: _textDirection,
        child: AppFlowyEditor(
          // widget pass through
          editable: widget.editable,
          shrinkWrap: widget.shrinkWrap,
          autoFocus: true,
          header: widget.header,
          // local states
          editorScrollController:
              widget.scrollController ?? editorScrollController,
          editorState: editorState,
          editorStyle: desktopEditorStyle(),
          footer: generateFooter(),
          blockComponentBuilders: _buildBlockComponentBuilders(),
          characterShortcutEvents: [
            ...standardCharacterShortcutEvents,
            if (roomId != null) ...mentionShortcuts(context, roomId),
          ],
          commandShortcutEvents: [...standardCommandShortcutEvents],
          disableAutoScroll: true,
        ),
      ),
    );
  }
  ```